### PR TITLE
New Port Destructivator

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Destructivator

## Game Information
- **Title**: Destructivator
- **URL**: https://pugfuglygames.itch.io/destructivator

### CFW Tests
CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720) 

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/destructivator/Destructivator.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then
```
Comment (Cebion) [inline ports/destructivator/Destructivator.sh:35]: Your game does not use gl4es (libGL.so.1) so this if libgl is not needed

File: ports/destructivator/Destructivator.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/destructivator"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (Cebion) [inline ports/destructivator/Destructivator.sh:31]: try to use the new logging method that overwrites itself
`> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1`

**Review by Cebion — APPROVED** [review]

**Review by Cebion — CHANGES_REQUESTED** [review]
See comments

File: ports/destructivator/Destructivator.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/destructivator/Destructivator.sh]: Okies

File: ports/destructivator/Destructivator.sh (hunk 2 above)
Comment (Fraxinus88) [reply ports/destructivator/Destructivator.sh]: Updated!

**Review by Cebion — APPROVED** [review]

**Review by Slayer366 — APPROVED** [review]

**Pull request merged**